cmd/bundle/main.go

```diff
@@ -96,6 +106,43 @@ func Do(ctx context.Context) error {
 	}
 
 	log.Printf("Image content was extracted to %s\n", flagValues.target)
+	if flagValues.showListing {
+		t := tablewriter.NewWriter(os.Stdout)
```

Comment:
Do we want some table headers?

Reply:
I tried to mimik the `ls -l` or `find -ls` look and feel and they don't come with headers, but I am not again it either. If you feel like a header would be helpful, I can add that.

Reply:
Yeah, I noticed your sample as well. Let's start without headers.

Reply:
I played around with headers, but they really make the output way wider than I like to be honest. Since `ls` also comes without headers and the output is very similar to the `-l` output of `ls`, I would argue with should also go without headers. What do you think?

Reply:
As written above, we start without headers.